src/devices/auriol_4ld5661.c

```diff
@@ -71,7 +76,8 @@ static int auriol_4ld5661_decode(r_device *decoder, bitbuffer_t *bitbuffer)
         data_t *data = data_make(
                 "model",            "Model",        DATA_STRING, "Auriol-4LD5661",
                 "id",               "ID",           DATA_FORMAT, "%02x", DATA_INT, id,
-                "battery_ok",       "Battery OK",   DATA_INT, batt_ok,
+                "channel",          "Channel",      DATA_INT,    channel,
+                "battery_ok",       "Battery",      DATA_INT,    !!battery,
```

Comment:
don't change

Reply:
Removed the channel part as we are not sure if thats really the channel or not